test/test_quantized.py

```diff
@@ -190,6 +190,31 @@ def test_qadd_relu_different_qparams(self):
         np.testing.assert_equal(qCrelu, qCrelu_hat.int_repr(),
                                 "Quantized addition with ReLU failed.")
 
+    """Tests max pool operation on quantized tensors."""
```

Comment:
In addition to these tests, please expose the op as part of nn.quantized.functional by modifying torch/nn/quantized/functional.py.
Also, add corresponding tests for the functional at: caffe2/test/test_nn_quantized.py

Reply:
Resolving this, as it's gonna be part of a different diff